role-make-version-changelog.sh

```diff
@@ -152,8 +152,14 @@ if [ "$skip" = false ]; then
               echo "--------------------"
               echo ""
               echo "REMOVE_ME: Recommend to itemize the change log entries in one of the following sections."
-              echo "REMOVE_ME: Use Other Changes for CI, testing, and other non-feature/non-bug related items."
-              echo "REMOVE_ME: If a section has no entries, remove the section."
+              echo "REMOVE_ME: If a section has no entries, it should be removed."
+              echo "REMOVE_ME: Use 'Other Changes' for non-feature/non-bug changes such as:"
+              echo "REMOVE_ME: - updates to documentation"
+              echo "REMOVE_ME: - updates to examples"
+              echo "REMOVE_ME: - deprecations (for example related to conscious language)"
+              echo "REMOVE_ME: For anything else, such as changes to tests or github actions,"
+              echo "REMOVE_ME: simply state '- no user-visible changes'"
```

Comment:
This sentence could be misunderstood since "- no user-visible changes" is meant to be add only if the changelog will be empty otherwise. Maybe write this like:

```suggestion
              echo "REMOVE_ME: Do not mention changes that not visible for users such as changes to tests or github actions."
              echo "REMOVE_ME: If you need to cut a release without any user-visible changes, simply state as only item '- no user-visible changes' instead of adding more details."
```